public/app/plugins/panel/datagrid/DataGridPanel.tsx

```diff
@@ -255,31 +259,31 @@ export function DataGridPanel({ options, data, id, fieldConfig, width, height }:
         smoothScrollX
         smoothScrollY
         overscrollY={50}
-        onCellEdited={isDatagridEditEnabled() ? onCellEdited : undefined}
-        getCellsForSelection={isDatagridEditEnabled() ? true : undefined}
-        showSearch={isDatagridEditEnabled() ? toggleSearch : false}
+        onCellEdited={isDatagridEnabled() ? onCellEdited : undefined}
```

Comment:
I'll leave these ff verifications here (and everywhere else) because even though the panel will be show at all now if the ff is not enabled, we will add an `enable editing` panel option soon after the G10 release which will give users the ability to enable/disable editing functionalities within the datagrid